media-video/openshot/openshot-2.4.0-r1.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_REQ_USE=xml
+PYTHON_COMPAT=( python3_{4,5,6} )
+DISTUTILS_SINGLE_IMPL=1
+
+inherit distutils-r1 gnome2-utils versionator xdg-utils
+
+MY_PN="${PN}-qt"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Free, open-source, non-linear video editor to create and edit videos and movies"
+HOMEPAGE="https://www.openshot.org/ https://launchpad.net/openshot"
+SRC_URI="https://launchpad.net/${PN}/$(get_version_component_range 1-2)/${PV}/+download/${MY_P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="1"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	dev-python/httplib2[${PYTHON_USEDEP}]
+	dev-python/PyQt5[${PYTHON_USEDEP},svg,webkit]
+	dev-python/pyzmq[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	>=media-libs/libopenshot-0.1.8[python,${PYTHON_USEDEP}]
+"
+DEPEND="
+	${PYTHON_DEPS}
```

Comment:
`distutils-r1` takes care of PYTHON_DEPS for you. Also, it should've been RDEPEND ;-).

Reply:
Okay, lets try that again. Im rebasing and hoping it goes well...

```diff
--- /usr/portage/media-video/openshot/openshot-2.4.0.ebuild	2018-03-10 09:08:28.641702227 +0200
+++ openshot-2.4.0-r1.ebuild	2018-04-29 11:44:28.448921711 +0300
@@ -13,7 +13,7 @@
 MY_P="${MY_PN}-${PV}"
 
 DESCRIPTION="Free, open-source, non-linear video editor to create and edit videos and movies"
-HOMEPAGE="http://www.openshot.org/ https://launchpad.net/openshot"
+HOMEPAGE="https://www.openshot.org/ https://launchpad.net/openshot"
 SRC_URI="https://launchpad.net/${PN}/$(get_version_component_range 1-2)/${PV}/+download/${MY_P}.tar.gz"
 
 LICENSE="GPL-3+"
@@ -21,10 +21,11 @@
 KEYWORDS="amd64 x86"
 
 RDEPEND="
-	dev-python/PyQt5[svg,webkit,${PYTHON_USEDEP}]
-	>=media-libs/libopenshot-0.1.8[python,${PYTHON_USEDEP}]
 	dev-python/httplib2[${PYTHON_USEDEP}]
+	dev-python/PyQt5[${PYTHON_USEDEP},svg,webkit]
 	dev-python/pyzmq[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	>=media-libs/libopenshot-0.1.8[python,${PYTHON_USEDEP}]
 "
 DEPEND="
 	dev-python/setuptools[${PYTHON_USEDEP}]
```

```
~/git/gentoo/media-video/openshot $ repoman full -dx .

RepoMan scours the neighborhood...
RepoMan sez: "If everyone were like you, I'd be out of business!"
```